types/params.go

```diff
@@ -36,7 +36,7 @@ func DefaultBlockParams() tmproto.BlockParams {
 	return tmproto.BlockParams{
 		MaxBytes:   22020096, // 21MB
 		MaxGas:     -1,
-		TimeIotaMs: 1000, // 1s
+		TimeIotaMs: 1000, // 1s, parameter is now unused
```

Comment:
Maybe we should change this to 1 millisecond, since that's what we're actually using. Doesn't matter for the behavior, of course, but might prevent confusion if someone were to look it up.